test/ObjectField_test.js

```diff
@@ -218,19 +218,6 @@ describe("ObjectField", () => {
       expect(labels).eql(["baz", "bar", "qux", "foo"]);
     });
 
-    it("should throw when order list contains an extraneous property", () => {
```

Comment:
Can you uncomment keep this test please and change the expectation ?

Reply:
Done